src/components/DateInput/styles.tsx

```diff
@@ -0,0 +1,50 @@
+import styled from 'styled-components';
+import { Open_Sans as OpenSans } from 'next/font/google';
+import COLORS from '@/styles/colors';
+import { H4 } from '@/styles/text';
+
+const openSans = OpenSans({ subsets: ['latin'] });
```

Comment:
note: we may want to import the `openSans` instance from a `fonts.ts` file in the future

Reply:
Note: done by jinkang, ty!